src/client/components/Agenda/WorkersCalendar.js

```diff
@@ -90,7 +90,7 @@ const StyledAvatar = styled(Avatar)``;
 
 const WorkerHeader = shouldUpdate(() => false)(({ worker }) => (
   <StyledWorkerHeader>
-    <StyledAvatar name={fullName(worker)} showTooltip color={worker.avatar.color} size={SMALL} />
+    <StyledAvatar name={fullName(worker)} to={`/person/${worker._id}`} showTooltip color={worker.avatar.color} size={SMALL} />
```

Comment:
Never hard code route but use getPathByName from route !!!!
so here: to={`${getPathByName('person')}/${worker._id}`}
eventually we can add params to getPathByName to write it like this
to={`${getPathByName('person', worker._id)}`} => better